doc/man3/SSL_CTX_set_options.pod

```diff
@@ -265,6 +265,20 @@ functionality is not required. Those applications can turn this feature off by
 setting this option. This is a server-side opton only. It is ignored by
 clients.
 
+=item SSL_OP_CLEANSE_PLAINTEXT
+
+By default TLS connections keep a copy of received plaintext
+application data in a static buffer until it is overwritten by the
+next portion of data. When enabling SSL_OP_CLEANSE_PLAINTEXT
+deciphered application data is cleansed by calling OPENSSL_cleanse(3)
+after passing data to the application. Data is also cleansed when
+releasing the connection (eg. L<SSL_free(3)>).
+
+Since OpenSSL only cleanses internal buffers, the application is still
+responsible for cleansing all other buffers. Most notable this applies
```

Comment:
@slontis made two comments about this. I think you missed the one "notable" -> "notably"

Reply:
ups sorry

Reply:
So @slontis also meant the comma after the notably.

I am not sure about this either.
What do you think?

Reply:
@slontis; sorry i misread your comment

fixed it as well

Reply:
I'm not sure about the comma - so whatever.